### PR TITLE
add git onboarding & adjust sidebar

### PR DESCRIPTION
**Pull request by chinathaip**
Group basic programming course documents to "Basic Programming" sidebar

Add another group for git documentation

Thank you kind sir

### CODE REVIEW

**Comment by chinathaip** [comment]
pls try run on your machine to see what I did. Really need second + third opinions on this

File: docs/git/git.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+![Untitled](/img/git/Untitled4.png)
+
+*NOTE: More details relating to git branches for collaborative work in our club will be covered later*
+
+### HEAD Pointer
+
+A HEAD Pointer represents the currently “checked-out” branch or commit in your repository. It essentially points to the tip of the branch or the specific commit that is currently being used as the basis for your working directory
+
+Example below shows that the HEAD pointer usually points to the latest commit in your project, but can also change when you “checkout” to a specific commits
```
Comment (Lxkas) [inline docs/git/git.md:51]: Where detached head explanation :3

**Review by Lxkas — APPROVED** [review]

**Comment by Lxkas** [comment]
Yea, tried it, look gucci to me
merge when ready

**Comment by chinathaip** [comment]
Aight merging

**Comment by anascence** [comment]
ahh i am late, sorry xD , just finished my class

**Pull request merged**